crates/swc_css_codegen/tests/fixture/comments/input.css

```diff
@@ -0,0 +1,7 @@
+/* Leading */
+
+a {
+    background: /* Leading */ red /* Trailing */;
+}
+
```

Comment:
Nice work! 

Can you add some comments in the function and in the at rule?

e.g.

```css
@media (width >/* foo */= 30em) {
 .test {
    width: calc(100%-/* foo */20px);
  }
}
```

Reply:
@yisibl Yes, I am still thinkig how better solve it, here just example how we should store them

Reply:
Can you resolve this issue?

Reply:
Yes, planned, want to finish some stuff before it